test/python/expected/upgrade_validation/expected_dependency.out

```diff
@@ -601,10 +628,19 @@ Function sys.sqlvariant_tinyint(sys.sql_variant)
 Function sys.sqlvariant_uniqueidentifier(sys.sql_variant)
 Function sys.sqlvariant_varchar(sys.sql_variant)
 Function sys.square(double precision)
+Function sys.st_transform(sys.geography)
+Function sys.stasbinary(sys.geography)
+Function sys.stasbinary(sys.geometry)
+Function sys.stastext(sys.geography)
+Function sys.stastext(sys.geometry)
+Function sys.stdistance(sys.geography,sys.geography)
+Function sys.stdistance(sys.geometry,sys.geometry)
```

Comment:
For all the these new functions, let's create couple of views in the prepare file which uses these functions so that we get good MVU coverage.

Reply:
Done